src/components/implementation/tests/bench_sched_yield/bench_sched_yield.c

```diff
@@ -17,36 +17,47 @@
 
 /* lo and hi is actually running at the same prio */
 #define ITERATION 10000
-/* #define PRINT_ALL */
+#define PRINT_ALL
 
 thdid_t yield_hi = 0, yield_lo = 0;
+
 volatile cycles_t start;
-volatile cycles_t end;
+volatile int count;
 
 struct perfdata perf;
 cycles_t result[ITERATION] = {0, };
 
 /***
- * We're measuring 2-way context switch time. 
+ * We're measuring one-way context switch time. 
  */
 void
 yield_hi_thd(void *d)
 {
-	/* Never stops running; low priority controls how many iters to run. */
-	while (1) {
+	cycles_t end;
+
+	while (count < ITERATION) {
 		debug("h1,");
```

Comment:
I know you didn't add this code, but there is no reason to keep meaningless printouts in the code. Feel free to remove them.